inspirehep/records/marshmallow/literature/common/author.py

```diff
@@ -39,3 +42,31 @@ def get_last_name(self, data):
             return names[0] or missing
 
         return missing
+
+
+class AuthorAutocompleteSchema(Schema):
+    input_field = fields.Method("generate_name_variations", dump_to="input")
```

Comment:
I'd put in all fields ``dump_only=True`` everywhere as we have in all serializers e

Reply:
Then why for example `AuthorSchemaV1` is missing it? :)

Reply:
That’s a mistake thrn, we have to add it everywhere. We’ll add it in a new PR will create a US

Reply:
:+1: